src/components/MatchInfo.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const searchQuery = styled.input`
+  border: red solid;
+`;
+class MatchInfo extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      searchQuery: '',
+      searchQueryValidate: false,
+      match: []
+    };
+  }
+
+  handleChange = event => {
+    this.setState({ searchQuery: event.target.value.trim() });
+  };
+
+  handleSubmit = () => {
+    this.queryValidate();
```

Comment:
no need for this function at all since validation will be 1-liner.
do not perform any validation on submit, rather do it directly on input change.